src/tracer.rs

```diff
@@ -325,15 +332,17 @@ pub fn generate_tracer_data(project: &Workspace, test: &Path, config: &Config) -
     if let Ok(obj) = OFile::parse(unsafe {file.as_slice() }) {
         let analysis = get_line_analysis(project, config); 
         let data = if obj.is_little_endian() {
-            get_line_addresses::<LittleEndian>(manifest, 
-                                               &obj, 
-                                               &analysis, 
-                                               config.ignore_tests)
+            get_line_addresses(LittleEndian,
```

Comment:
Could something like an if let be used to set an endianness variable and then `get_line_addresses(..).map_err(..)` called at the end once. Just a small thing a part from that I'm happy :smile:

Reply:
It seems it is not possible with this version of gimli. Endianity is used as a generic parameter, and, unfortunately, cannot be boxed.

Reply:
So just to double check this code isn't possible?
```
let endianity = if obj.is_little_endian() {
    LittleEndian
} else {
    BigEndian
}
get_line_addresses(LittleEndian,
                   manifest,
                   &obj,
                   &analysis
                   config.ignore_tests).map_err(..)
'''

Reply:
no, it is not :(

Reply:
I guess it makes sense given how the previous version of gimli worked. But in that case everything's great and I'll merge this now, thanks a ton :+1:

Reply:
It's not possible because `LittleEndian` and `BigEndian` are different types. You could consider using `RunTimeEndian` instead. That gives simpler and smaller code but a minor runtime performance loss. I doubt parsing dwarf is a major part of the runtime of tarpaulin though.

Reply:
It probably isn't but I don't think the readability gain is large enough to warrant changing it personally. I'd rather have a bit extra speed

Reply:
@philipc Thanks! I've opened corresponding PR https://github.com/xd009642/tarpaulin/pull/52